Assets/Prefabs/Object/Astronaut.prefab

```diff
@@ -97,8 +97,8 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 3e8cea98d335472ea8eaf2963c03c09e, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  minMoveSpeed: 0.2
-  maxMoveSpeed: 0.4
```

Comment:
![ask](https://img.shields.io/badge/review-ask-9b71ae.svg) 
Is this change intended? I think we decided to go with this number 0.2 - 0.4.
(But of course we can make changes after this PR merged)

Reply:
in PR-#26 https://github.com/shogo0x2e/BlackHoleBoogie/pull/26/files, I removed the line ```moveSpeed = moveSpeed + 0.01F; // A bit of acceleration``` (I don't know who added that), and I believe it's this line that was making objects very fast. So I increased their speed a little, but yeah we can change after this PR if it's too fast